src/Lecture4.hs

```diff
@@ -226,7 +280,10 @@ implement the next task.
 -}
 
 combineRows :: NonEmpty Row -> Stats
-combineRows = error "TODO"
+combineRows (row:|rows) = foldl1' (<>) $ Prelude.map rowToStats (row:rows)
+--combineRows rows = sconcat $ Data.List.NonEmpty.map rowToStats rows
+-- is there a way to not need to qualify this map to disambiguate? I thought the
+-- types would make it obvious.
```

Comment:
Haskell doesn't have name overloading. I.e. it's not possible to have two functions with the same name but different types in scope.

However, you can use `fmap` here because `NonEmpty` implements `Functor` instance 🙂